### DISCUSSION TITLE
Allow lists of tasks to have the option of "convert to issue" from project view.

### DISCUSSION THREAD

**Discussion by mjervelund** [community/community#113171] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

When working with GitHub issues, you can create a list of tasks within an issue by using the following format:

```markdown
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3
```
However, this functionality is currently limited to the issue view (e.g., gh.com/owner/repo/issues/234234). Unfortunately, it is not available when viewing the project as a whole.

It would be beneficial to have a consistent experience across both the issue and project views, allowing users to create tasks directly from a list within the project view.

![image](https://github.com/community/community/assets/13621747/f9a398fa-74b4-4e1e-816a-bd3cd2cff5f0)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by lin20260407** [comment, marked as answer]
Hello,

Thank you for taking the time to share your feedback on the Projects feature within GitHub. We're always looking for ways to improve the experience for our users, and insights like yours are invaluable in guiding those enhancements.

You've highlighted an important aspect of task management within GitHub issues and Projects—specifically, the ability to create and manage task lists directly from the project view. This suggestion for a more integrated and seamless experience across both issues and project views is indeed a compelling one. It recognizes the need for consistency in how tasks are handled, regardless of where they're being viewed or managed.

The functionality to create tasks within an issue using the - [ ] Task format has proven to be a highly effective way to organize work and track progress. Extending this capability to the project view could enhance the workflow significantly, making it easier for teams to manage their tasks and priorities directly within the project context.

While I can't offer an immediate solution or timeline for when such a feature might be implemented, I want to assure you that your feedback has been forwarded to our product development team for consideration. We understand the importance of continually evolving our platform to meet the needs of our community, and suggestions from users like you play a key role in that process.

In the meantime, we encourage you to explore existing project management tools within GitHub Projects, such as labels, milestones, and automated kanban boards, which might offer alternative ways to organize and prioritize your work.

We appreciate your contribution to making GitHub a better platform for developers and teams. Please keep the feedback coming, and don't hesitate to reach out if you have more ideas, questions, or need assistance with any of our features.

**Comment by firuzaharun** [comment]
agree, why it is not in Project view :(
